core/tauri-runtime-wry/Cargo.toml

```
@@ -13,12 +13,13 @@ exclude = [ "CHANGELOG.md", "/target" ]
 readme = "README.md"
 
 [dependencies]
-wry = { version = "0.24.4", default-features = false, features = [ "file-drop", "protocol" ] }
+wry = { git = "https://github.com/crabnebula-dev/wry", branch = "feat/0.24-tracing", default-features = false, features = [ "file-drop", "protocol" ] }
```

```suggestion
wry = { git = "https://github.com/tauri-apps/wry", default-features = false, features = [ "file-drop", "protocol" ] }
```

This shouldn't point to the fork 😉

---

it has to point to the fork until it's published

---

Then lets merge the wry PR first, it should be easier to merge that one anyway